### PR TITLE
ghosts no longer hear clockcult sounds

### PR DESCRIPTION
**Pull request by optimumtact**
They're incredibly annoying as an admin observer
:cl: oranges
sounddel: ghosts no longer hear clockcult sounds
/:cl:

### CODE REVIEW

**Comment by Bawhoppen** [comment]
Can't you just mute your game in windows volume mixer?

File: code/modules/antagonists/clockcult/clock_structures/eminence_spire.dm [hunk 1]
```diff
@@ -106,7 +106,7 @@
 	if(ismob(eminence_nominee))
 		if(!eminence_nominee.client || !eminence_nominee.mind)
 			hierophant_message("<span class='brass'><b>[eminence_nominee] somehow lost their sentience!</b> The eminence spire has been reset.</span>")
-			for(var/mob/M in servants_and_ghosts())
+			for(var/mob/M in SSticker.mode.servants_of_ratvar)
```
Comment (optimumtact) [inline code/modules/antagonists/clockcult/clock_structures/eminence_spire.dm]: whoops I need to retain it for this since it polls the ghosts

**Comment by optimumtact** [comment]
Yes I can just disable my game sound so I can no longer hear adminhelps great fucking idea baw wow you're so clever and smart if only I had thought of that

File: code/modules/antagonists/clockcult/clock_mobs/_eminence.dm [hunk 2]
```diff
@@ -90,7 +90,7 @@
 		visible_message("<span class='brass'><b>You feel light slam into your mind and form words:</b> \"[capitalize(message)]\"</span>")
 		playsound(src, 'sound/machines/clockcult/ark_scream.ogg', 50, FALSE)
 	message = "<span class='big brass'><b>The [GLOB.ratvar_awakens ? "Radiance" : "Eminence"]:</b> \"[message]\"</span>"
-	for(var/mob/M in servants_and_ghosts())
+	for(var/mob/M in SSticker.mode.servants_of_ratvar)
 		if(isobserver(M))
```
Comment (ChangelingRain) [inline code/modules/antagonists/clockcult/clock_mobs/_eminence.dm]: ?

File: code/modules/antagonists/clockcult/clock_mobs/_eminence.dm [hunk 3]
```diff
@@ -189,12 +189,12 @@
 			command_text = "The Eminence orders that defenses should be built starting from the top of Reebe!"
 	if(marker_icon)
 		new/obj/effect/temp_visual/ratvar/command_point(get_turf(A), marker_icon)
-		for(var/mob/M in servants_and_ghosts())
+		for(var/mob/M in SSticker.mode.servants_of_ratvar)
 			to_chat(M, "<span class='large_brass'>[replacetext(command_text, "GETDIR", dir2text(get_dir(M, command_location)))]</span>")
```
Comment (ChangelingRain) [inline code/modules/antagonists/clockcult/clock_mobs/_eminence.dm]: this should probably be seen by ghosts in some manner

**Comment by Bawhoppen** [comment]
Maybe, I don't want my view of the game as observer comprised because you get annoyed by some sounds though

File: code/modules/antagonists/clockcult/clock_mobs/_eminence.dm (hunk 3 above)
Comment (optimumtact) [reply code/modules/antagonists/clockcult/clock_mobs/_eminence.dm]: I don't think it's necessary, but I can readd it for this, but just remove the sound for ghosts

**Comment by Kromgar** [comment]
The whole round is an annoying jumble of sounds that make me die a little inside more and more

**Comment by YakumoChen** [comment]
can't you just disable ambience or whatever so it does mute your game sounds but you can still hear admin bwoinks

**Comment by ExcessiveUseOfCobblestone** [comment]
Every spell having a sound louder than oranges under the covers is really obnoxious as well

**Comment by PKPenguin321** [comment]
I like this because I don't want my view of the game as observer comprised by CLANG CLANG CLONG CRRRANK DONGDONGDONG CLUNK CRRRRKRKRKRKK CRUNK TIKTOKTIKTOKTIKTOKTIKTOK WHRRRRRRRRRRRRRRRR CLONG TNKTNKTNKTNKTNKTNK FSHHHHHHHHHHHHHHHH CLONG CLONG CLANG

**Comment by Xhuis** [comment]
When I asked which sounds were the problem sounds nobody answered me.

**Comment by Durkel** [comment]
We didn't want to hurt your feelings by saying that it was all of them.

**Comment by Bawhoppen** [comment]
I mean I don't care that much, just seems like this is an extreme reaction

**Comment by juanototo2** [comment]
P r e f e r e n c e s

**Comment by Potato-Masher** [comment]
I'm sorry, but...

>P r e f e r e n c e s

**Pull request closed without merging**